core/src/integration-test/java/com/scalar/db/storage/StorageSingleClusteringKeyScanIntegrationTestBase.java

```diff
@@ -0,0 +1,675 @@
+package com.scalar.db.storage;
```

Comment:
This is the main class in this PR and it's an integration test of scan operation for single clustering key table.